docs/source/api_reference/forecasting.rst

```diff
@@ -439,6 +439,14 @@ Deep learning based forecasters
 
     HFTransformersForecaster
 
+.. currentmodule:: sktime.forecasting.ttm
```

Comment:
introduce a separate category maybe, or split?

There are enough DL forecasters to split them, high-level we have 1st generation recurrence architectures, 2nd generation transformer architectures, and 3rd generation zero-shot/few-shot transformers, aka foundation models.

Not a blocker, but I think for user experience might be worth considering. FYI @benHeid, how would you split this?

Reply:
Just sketching some ideas
- MLP
	- LTSFLinearForecaster
	- LTSFDLinearForecaster
	- LTSFNLinearForecaster

- Recurrent
	- NeuralForecastRNN
	- NeuralForecastLSTM

- Transformer / Pre-trained
	- HFTransformersForecaster
	- TinyTimeMixerForecaster
	- LTSFTransformerForecaster

- Foundation / Pre-trained
	- TimesFM

- not sure about how they work
	- PyKANForecaster
	- PytorchForecastingTFT
	- PytorchForecastingNBeats
	- CINNForecaster